ast/Trees.h

```diff
@@ -1111,6 +1112,25 @@ EXPRESSION(InsSeq) {
 };
 CheckSize(InsSeq, 56, 8);
 
+EXPRESSION(RuntimeMethodDefinition) {
+public:
+    // This should be equivalent to MethodDef.declLoc.
```

Comment:
```suggestion
    // This should be equivalent to MethodDef.declLoc.
    // It's called `loc` because every node in the AST has to have a `loc` field.
```